tests/e2e/files/devfiles/plugins/TypescriptNodeDebug2PluginTest.yaml

```diff
@@ -27,7 +27,7 @@ commands:
     actions:
       - workdir: '${CHE_PROJECTS_ROOT}/nodejs-web-app/app'
         type: exec
-        command: npm install && nodemon --inspect app.js
+        command: npm ci && nodemon --inspect app.js
```

Comment:
Original issue was about using `npm ci` command in Eclipse Che E2E test typescript project.

This is another typescript project: "**nodejs-web-app**".
So, I don't think we should change test projects.

Reply:
Okay, I will remove the changes

Reply:
Done